localbtfs/localbtfs.go

```diff
@@ -77,6 +84,14 @@ func NewNode(dir string, attrs map[string]string) (testbedi.Core, error) {
 		}
 	}
 
+        if remoteapiaddrstr, ok := attrs["remoteapiaddr"]; ok {
```

Comment:
@richlee33 this file is missing. Could you try from root folder `go fmt ./...` ?

Reply:
sorry, i'll do better next time.